### DISCUSSION TITLE
when adopt  remote ray cluster, how the `rayWorker` read the local files

### DISCUSSION THREAD

**Discussion by vincent-pli** [data-prep-kit/data-prep-kit#833] [Q&A] [answered]
I means if I launch to remote ray cluster and want it handle the file on my laptop, is that possible?

**Comment by agoyal26** [comment]
@roytman @touma-I Please see ^^

**Comment by roytman** [comment, marked as answer]
To access files locally on a remote Ray cluster, the files must be accessible within the Ray image. They can either be mounted or copied. 
Another option is to run an S3 server on your laptop, such as Minio or "localstack".  

@blublinsky can provide more details—he’s the Ray expert.

**Reply by vincent-pli** [reply]
Great thanks, 
BTW, I have another question, still about Ray.
I saw this in `data-processing-lib/ray/src/data_processing_ray/runtime/ray/transform_launcher.py`
```
ray.init(f"ray://localhost:10001", ignore_reinit_error=True)
```
the `10001` is for ray client, that's means we interact with ray cluster with ray client, but ray dashboard cannot get logs from job which launched by ray client, only support job submission.

So, how do we handle the logs, if it runs on a remote worker of ray? thanks
@blublinsky

**Reply by shivdeep-singh-ibm** [reply]
In usual practice, we don't interact with the ray server/head using client. The preferable way is login to ray server/head node and 
run the commandline application or use python/jupyter to run the transform.
